sites/page-details/index.json-ld

```diff
@@ -265,15 +274,84 @@
 					"Déplacez le composant de date de modification hors de la grille et insérez-le juste avant la balise de fermeture <code>&lt;section></code>. Supprimez le <code>&lt;div class=\"col-xs-12\"></code> dans lequel il se trouvait."
 				]
 			}
+		},
+		{
+			"@id": "_:implement_pd_4to5",
+			"iteration": "_:iteration_pd_5",
+			"name": {
+				"en": "Migration from version 3.1 to 3.2",
+				"fr": "Migration de la version 3.1 vers 3.2"
+			},
+			"introduction": {
+				"en": "This implementation is meant for developers or publishers trying to migrate their page from using the page details version 3.1 to 3.2 (removal of \"col-lg-9\" class).",
+				"fr": "Cette implémentation est destinée aux développeurs ou aux éditeurs qui tentent de migrer leur page de la version 3.1 vers la version 3.2 (suppression de la classe « col-lg-9 »)."
+			},
+			"instructions": {
+				"en": [
+					"Remove the \"col-lg-9\" class from the <code>&lt;div class="col-sm-8 col-md-9 col-lg-9"&gt;</code> element."
```

Comment:
The build is currently failing due to the following jsonlint warning at the start of ``<code>``:
```
Running "jsonlint:all" (jsonlint) task
>> File "sites/page-details/index.json-ld" failed JSON validation at line 291, column 67.
...code>&lt;div class="col-sm-8 col-md-9 co...
-----------------------^
Unexpected token c
Warning: 1 file failed validation Use --force to continue.
```

Not sure what the deal is :S. Some pre-existing instruction strings already contain ``<code>`` tags without a problem (case in point: https://github.com/wet-boew/GCWeb/blob/b6388a131e52907e153f27a64763d50160f12ecc/sites/page-details/index.json-ld#L260).

Reply:
Fixed it - forgot to escape a couple of double-quotes later on in the line. Guess I was overly-fixated on the earlier column jsonlint was complaining about :D.